tests/baselines/reference/jsDeclarationsExportDefinePropertyEmit.js

```diff
@@ -139,7 +139,7 @@ export namespace f {
      * @template T
      * @param {T} a
      */
-    export function self<T>(a: T): T;
+    function self<T>(a: T): T;

```

Comment:
While working on this, I found a need for the js declaration emitter to automatically add/strip export modifiers from nested namespace members as needed; so the result of this can be seen in some baseline updates like this - since every member of the containing namespace was `export`ed, there was no need to explicitly have the `export` modifiers. Now they get stripped, consistent with our transform-based declaration emitter.